### DISCUSSION TITLE
WebHooks registering

### DISCUSSION THREAD

**Discussion by TheButor** [capcom6/android-sms-gateway#135] [Q&A] [answered]
Hi,
I'm implementing this SMS Gateway WEBHOOKS REGISTRATION but have troubles to understand what are:
- unique-id
- uuid
In the curl request to register to the webhooks.
![image](https://github.com/user-attachments/assets/eb1728ce-6d8a-4b76-bee5-bfe5bc61bc8a)

Can someone help me handling it ?
Thanks in advance

**Comment by capcom6** [comment, marked as answer]
Hello!

1. `<unique-id>`: This is used to identify the webhook within your account. You can:
   - Replace it with any unique value of your choice.
   - Leave it empty, and the server will generate an ID for you.

2. `<your-uuid>`: This refers to the unique identifier provided by webhook.site, a service used for testing and debugging webhooks. You'll receive this UUID when you first visit webhook.site.

The example in the documentation demonstrates how to use webhook.site for receiving and testing webhooks. However, in a production environment, you would replace this with your own endpoint URL.

**Reply by TheButor** [reply]
Thanks a lot !
